### PR TITLE
RHD-1258 /register allways redirects to production

### PR DESCRIPTION
**Pull request by paulrobinson**

### CODE REVIEW

File: register.html.slim [hunk 1]
```diff
@@ -8,7 +8,7 @@ head
       clientId: 'web'
     });
 
-    var redirectUri = 'http://developers.redhat.com/confirmation';
+    var redirectUri = app.ssoConfig.confirmation;
```
Comment (paulrobinson) [inline register.html.slim]: @wesbos Any ideas on how I get the variables into this page? Currently I'm seeing 'Uncaught ReferenceError: app is not defined' when visiting this page.

I need to ensure the right KeyCloak server is used in staging.

I also need to paramaterise line 6 above (url: "https://developers.redhat.com/auth",)

Comment (wesbos) [inline register.html.slim]: Are you loading all of our JS on that page?

Comment (wesbos) [inline register.html.slim]: I see - let me fork this and see if we can just pull in those bits.

Comment (wesbos) [inline register.html.slim]: pull this into yours https://github.com/redhat-developer/developers.redhat.com/pull/639

Comment (paulrobinson) [inline register.html.slim]: I now get 'Uncaught ReferenceError: app is not defined'. Any ideas?

**Comment by paulrobinson** [comment]
@wesbos can you take a look at why this is failing? See my comments above.

File: register.html.slim (hunk 1 above)
Comment (wesbos) [inline register.html.slim]: @paulrobinson You are getting this on staging?

**Comment by wesbos** [comment]
@paulrobinson Can you cherry pick this into yours:

https://github.com/redhat-developer/developers.redhat.com/commit/34767ae777fdb9230ff10dad0bdbd6e114c20a81

**Comment by paulrobinson** [comment]
Thanks. I wonder why I didn't do it that way the first time? Hopefully there's not something we are missing. I'll take a look.

**Comment by ollyjshaw** [comment]
Needs a rebase

**Comment by paulrobinson** [comment]
Replaced by https://github.com/redhat-developer/developers.redhat.com/pull/664

**Pull request closed without merging**